index.js

```diff
@@ -129,91 +102,80 @@ const HOME_MENU_BUTTONS_ARRAY = [
     {
         id: ELEMENT_IDS.digitalProgrammeListTab,
         innerText: "Digital Programme List",
-        onclick: renderDigitalProgrammeList
+        onclick: () => console.log(innerText)
     },
     {
         id: ELEMENT_IDS.digitalEpgTab,
         innerText: "Digital EPG",
-        onclick: renderDigitalEpg
+        onclick: () => console.log(innerText)
     },
     {
         id: ELEMENT_IDS.applicationsTab,
-        param2: "applications",
+        buttonName: "applications",
         innerText: "Applications",
         options: [
             {
-                id: ELEMENT_IDS.photoSharingPlus,
                 innerText: "Photo Sharing Plus",
                 onclick: () => console.log("Photo Sharing Plus"),
             },
             {
-                id: ELEMENT_IDS.photoFrameMode,
                 innerText: "Photo Frame Mode",
                 onclick: () => console.log("Photo Frame Mode"),
             },
         ]
     },
     {
         id: ELEMENT_IDS.recordingsTab,
-        param2: "recordings",
+        buttonName: "recordings",
         innerText: "Recordings",
         options: [
             {
-                id: ELEMENT_IDS.titleList,
                 innerText: "Title List",
                 onclick: () => console.log("Title List")
             },
             {
-                id: ELEMENT_IDS.timerList,
                 innerText: "Timer List",
                 onclick: () => console.log("Timer List"),
             },
             {
-                id: ELEMENT_IDS.errorList,
                 innerText: "Error List",
                 onclick: () => console.log("Error List"),
             },
             {
-                id: ELEMENT_IDS.manualTimerRec,
                 innerText: "Manual Timer REC",
                 onclick: () => console.log("Manual Timer REC"),
             },
         ]
     },
     {
         id: ELEMENT_IDS.mediaTab,
-        param2: "media",
+        buttonName: "media",
         innerText: "Media",
         options: [
             {
-                id: ELEMENT_IDS.photo,
                 innerText: "Photo",
                 onclick: () => console.log("Photo"),
             },
             {
-                id: ELEMENT_IDS.music,
                 innerText: "Music",
                 onclick: () => console.log("Music"),
             },
             {
-                id: ELEMENT_IDS.video,
                 innerText: "Video",
                 onclick: () => console.log("Video")
             },
         ]
     },
     {
         id: ELEMENT_IDS.settingsTab,
-        param2: "settings",
+        buttonName: "settings",
```

Comment:
Instead of introducing new key `innerText` could be transformed

Reply:
Don't really know how you mean. I need buttonName to handle disable buttons.